pages/dao/[dao]/drafts/[draft]/index.ts

```diff
@@ -24,10 +24,15 @@ export const getServerSideProps: GetServerSideProps = async ({
   const daoContext = await getDaoContext(accountId, daoId as string);
 
   const dao = daoContext?.dao;
+  let draft;
 
-  const draft = await draftService.getDraft(draftId, dao, accountId);
+  try {
```

Comment:
Is there sense to place this try-catch block inside of the `getDraft` method?

Reply:
fixed